### PR TITLE
update nethermind's docker instruction

### PR DESCRIPTION
**Pull request by matilote**

### CODE REVIEW

File: from_docker.md [hunk 1]
```diff
@@ -106,19 +106,25 @@ docker run \
...
+python generate_eth1_nethermind_conf.py > "$TESTNET_NAME/public/eth1_nethermind_config.json"
+```
 
 #### Running:
 
 Note: the nethermind docker cannot handle user changes (error on p2p key write, permissions problem), the container runs nethermind root internally.
 ```shell
 docker run \
+  -p 8545:8545 \
+  -p 8546:8546 \
```
Comment (protolambda) [inline from_docker.md]:
```suggestion
  --net host \
```

File: from_docker.md [hunk 2]
```diff
@@ -106,19 +106,25 @@ docker run \
...
   --name nethermind0 \
-  -v ${PWD}/$TESTNET_NAME/public/eth1_nethermind_config.json:/networkdata/eth1_nethermind_config.json \
+  -v ${PWD}/$TESTNET_NAME/public/eth1_nethermind_config.json:/nethermind/chainspec/eth1_nethermind_config.json \
```
Comment (protolambda) [inline from_docker.md]: Is this really necessary, to overwrite the default with the mount? We specify the `/networkdata/....` path explicitly later in the arguments, and it works fine.

Comment (matilote) [reply from_docker.md]: should be fine with `networkdata`, reverted the change

**Review by protolambda — APPROVED** [review]
LGTM. Thank you!

File: from_docker.md (hunk 2 above)
Comment (LukaszRozmej) [reply from_docker.md]: no its not, you can reverse it

**Pull request merged**